CHANGELOG.md

```diff
@@ -1,18 +1,24 @@
 # Eclipse GLSP Server Node Changelog
 
-## v0.10.0- Upcoming
+## v1.0.0- Upcoming
 
 Inception of the Node GLSP Server.
 This project provides the Node based server component for the Eclipse Graphical Language Platform (GLSP).
 The implementation of this server is aligned with the default Java based [GLSP Server](https://github.com/eclipse-glsp/glsp-server).
-The initial initial implementation was contributed on behalf of STMicroelectronics.
+The [initial implementation](https://github.com/eclipse-glsp/glsp-server-node/commit/4fba8e8beef07798a7eff27c9c04ca68583e5960) has been contributed on behalf of STMicroelectronics.
```

Comment:
```suggestion
The [initial implementation](https://github.com/eclipse-glsp/glsp-server-node/commit/4fba8e8beef07798a7eff27c9c04ca68583e5960) was contributed on behalf of STMicroelectronics.
```
Nitpick, but i think the process is over, hence `was` is more suited than `has been`.